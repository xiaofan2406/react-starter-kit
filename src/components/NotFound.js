import React from 'react';
import { Link } from '@reach/router';

const NotFound = () => (
  <div>
    404 Nothing here. Go to <Link to="/">homepage</Link>
  </div>
);

export default NotFound;
