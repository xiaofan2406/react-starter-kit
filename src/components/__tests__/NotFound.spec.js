import React from 'react';
import { render } from 'react-testing-library';
import NotFound from '../NotFound';

test('NotFound component matches snapshot', () => {
  const { container } = render(<NotFound />);

  expect(container.firstChild).toMatchSnapshot();
});
