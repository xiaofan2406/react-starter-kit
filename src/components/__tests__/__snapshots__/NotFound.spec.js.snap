// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component matches snapshot 1`] = `
<div>
  404 Nothing here. Go to 
  <a
    aria-current="page"
    href="/"
  >
    homepage
  </a>
</div>
`;
